cosmos/dbt/selector.py

```diff
@@ -175,7 +175,7 @@ def filter_nodes(self, nodes: dict[str, DbtNode]) -> set[str]:
         # Index nodes by name, we can improve performance by doing this once
         # for multiple GraphSelectors
         if PATH_SELECTOR in self.node_name:
-            path_selection = self.node_name[len(PATH_SELECTOR) :]
+            path_selection = self.node_name[len(PATH_SELECTOR) :].rstrip("*")
```

Comment:
Just for my understanding, why does the path contain `*` regex matching?

Reply:
I think it's more the case that users may specify a `*` in the path selector and we want to support that

Reply:
Since we're checking if the path defined in the selector is "in" the dbt resource path string, the path stops working if users happen to pass * in Cosmos. Since this is a common practice when using a few dbt filters. and by removing `*` we seem to have the same/expected behaviour, I introduced this change.